api.swagger.yaml

```diff
@@ -6150,6 +6271,25 @@ components:
             enum:
               - Organisational user
               - Webmaster
+              - Author
+              - Moderator
+              - Editor
+              - Publisher
+              - Content reviewer
+              - Section manager
+              - Previewer
+              - Redacted
+              - Ghostwriter
+              - Brand manager
+              - Blogger
+              - Ideas administrator
+              - OU administrator
+              - Location admin
+              - Content layout manager
+              - External user
+              - Invotra support user
```

Comment:
Remove 'Invotra support user' and 'Services API user'/'Ghostwriter' as customers should not be able to set these.

Reply:
Removed the 3 roles